org.quetoo.Quetoo.appdata.xml

```diff
@@ -43,6 +43,6 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
-    <release version="0.1.0" date="2021-04-08" type="development"/>
+    <release version="0.1.0" date="2022-04-08" type="development"/>
```

Comment:
The release date is based on the date of the last commit. Not sure about the version number though.